user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/EditBatchDialog.java

```diff
@@ -137,9 +137,24 @@ private void onBatchNameChanged(
   }
 
   private void onRemoveLastRowClicked(ClickEvent<Button> clickEvent) {
+    SampleInfo sampleInfo = spreadsheet.getLastRowData();
+    fireEvent(new RemoveRowEvent(this, clickEvent.isFromClient(),
+        sampleInfo));
+  }
+
+  /**
+   * Removes the last row. Any checks for validity of the operation should be performed before using
+   * the related event
+   */
+  public void removeRow() {
```

Comment:
This name does not really fit. You are specifically removing the last row. As there will probably be a need to remove a specific row I would ask you to rename this method.

Reply:
has been renamed